staging/src/k8s.io/kubectl/pkg/cmd/create/create_namespace.go

```diff
@@ -36,16 +44,37 @@ var (
 	  kubectl create namespace my-namespace`))
 )
 
-// NamespaceOpts is the options for 'create namespace' sub command
-type NamespaceOpts struct {
-	CreateSubcommandOptions *CreateSubcommandOptions
+// NamespaceOptions is the options for 'create namespace' sub command
+type NamespaceOptions struct {
+	// PrintFlags holds options necessary for obtaining a printer
+	PrintFlags *genericclioptions.PrintFlags
+	// Name of resource being created
+	Name string
+	// StructuredGenerator is the resource generator for the object being created
```

Comment:
This command is not needed any more :)

Reply:
I guess he only forgot to remove the comment here :D

But yeah, please remove the comment!